src/devices/cpu/axc51/axc51.cpp

```diff
@@ -28,9 +28,15 @@
 #include "axc51.h"
 #include "axc51dasm.h"
 
-#define VERBOSE 1
+#define LOG_GENERAL  (1U <<  1)
```

Comment:
`LOG_GENERAL` is defined in logmacro.h itself and used for the abbreviated `LOG` (rather than `LOGMASKED`) macro.  It’s probably better not to redefine it unnecessarily.

Reply:
This anti-pattern is used all over the codebase, probably by developers who also didn't know any better. Unfortunately, now that code exists as a poor example to learn from. Please file a Github issue about it, assign it to me, and I'll file a PR by the end of the weekend.